docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -109,19 +109,32 @@ other types will be added, for example `regression`.
 
 An {oldetection} configuration object has the following properties:
 
-`n_neighbors`::
-  (integer) Defines the value for how many nearest neighbors each method of 
-  {oldetection} will use to calculate its {olscore}. When the value is 
-  not set, the system will dynamically detect an appropriate value.
+`compute_feature_influence`::
+  (boolean) If `true`, the feature influence calculation is enabled. Defaults to 
+  `true`.
+  
+`feature_influence_threshold`:: 
+  (double) The minimum {olscore} that a document needs to have in order to 
+  calculate its {fiscore}. Value range: 0-1 (`0.1` by default).
 
 `method`::
   (string) Sets the method that {oldetection} uses. If the method is not set 
   {oldetection} uses an ensemble of different methods and normalises and 
   combines their individual {olscores} to obtain the overall {olscore}. We 
   recommend to use the ensemble method. Available methods are `lof`, `ldof`, 
   `distance_kth_nn`, `distance_knn`.
-
-`feature_influence_threshold`:: 
-  (double) The minimum {olscore} that a document needs to have in order to 
-  calculate its {fiscore}. 
-  Value range: 0-1 (`0.1` by default).
\ No newline at end of file
+  
+`n_neighbors`::
+  (integer) Defines the value for how many nearest neighbors each method of 
+  {oldetection} will use to calculate its {olscore}. When the value is 
+  not set, the system will dynamically detect an appropriate value.
+  
+`outlier_fraction`::
+  (double) Sets the proportion of data that is assumed to be outlier prior to the 
+  {oldetection}. For example, 0.05 means, it is assumed that 5% of values are real outliers 
+  and 95% are inliers.
+  
+`standardize_columns`::
+  (boolean) If `true`, then the following operation is performed on the columns 
```

Comment:
One option here would be just to refer to wiki for this, i.e. this link https://en.wikipedia.org/wiki/Feature_scaling#Standardization_(Z-score_Normalization)